src/SourceBuild/content/repo-projects/source-build-externals.proj

```diff
@@ -1,2 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!--
+      We need to continue to use inner-clone for this repo, as there are hundreds of modified files
+      after a build, due to numerous patches and modifications we need to do.
+
+      Modified files make developer experience worse, as they would need to be manually reverted.
+    -->
+    <BuildArgs>$(BuildArgs) /p:UseInnerClone=true</BuildArgs>
```

Comment:
Specify this as an msbuild property here instead so that the `CopyRepoArtifacts` target can condition on it not being true. Move this BuildArgs line to the top of repo-projects/Directory.Build.targets and condition it on `UseInnerClone == true`.